src/app/components/fluid-grid/index.html

```diff
@@ -9,6 +9,34 @@
     The fluid grid component provides a responsive 12-column layout to organize content for all device sizes. The fluid grid consists of the <sky-code>sky-fluid-grid</sky-code>, <sky-code>sky-row</sky-code>, and <sky-code>sky-column</sky-code> tags. You place <sky-code>sky-column</sky-code> elements in <sky-code>sky-row</sky-code> elements, and then you can specify how much of the layout to use for each <sky-code>sky-column</sky-code> element on extra-small, small, medium, and large screens. The <sky-code>sky-fluid-grid</sky-code> parent element ensures consistent styling for all fluid grid instances.
   </sky-demo-page-summary>
 
+  <sky-demo-page-properties sectionHeading="Fluid grid properties">
+    <sky-demo-page-property
+      propertyName="disableMargin"
+      defaultValue="false"
+      isOptional="true"
+    >
+      Disables the outer left and right margin of the fluid grid container. This property accepts <sky-code>boolean</sky-code> values.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="gutterSize"
+      defaultValue="SkyFluidGridGutterSize.Large"
+      isOptional="true"
+    >
+      Specifies a <sky-code>SkyFluidGridGutterSize</sky-code> enum that defines the size of the padding between each column. The valid options are:
```

Comment:
It's superminor, but I'd change "that defines" to "to define" and change "each column" to "columns."

Also, do we really need the bullet list? If we really want the values to stand out, the list works. But we don't usually list values this way. Generally, we just list them in a sentence in the description. So in this case, I'm thinking something like: "The valid options are `Small` for a 10px gutter, `Medium` for a 20px gutter, and `Large` for a 30px gutter." ... We can keep the list if you prefer, but I don't think we really need the values to stand out that much.

Reply:
I just grabbed the list from another component. I like the sentence much better. I'll make that change.

Reply:
done.

Reply:
I updated this to use `sky-code` elements instead of tick marks. Otherwise, LGTM.